tests/scalers/azure-queue-pause.test.ts

```diff
@@ -60,6 +60,25 @@ test.serial(
   }
 )
 
+test.serial.cb(
```

Comment:
Due to it does the scaling to zero and then again to 1, maybe we should add a small delay here to ensure that the "possible" scaling out has happened.
WDYT?

Reply:
Agreed, goo idea!

Reply:
I'm sorry, I don't follow, do you mean that there should be delay after we check that deployment remains at 0? I don't see where exactly are we scaling out to 1 after this step

Reply:
No no, I said to ensure the bug is solved.
Right now, if you apply the annotation to pause the so with 0, the replica count goes to zero for a few seconds. 
My only concern is if this few seconds are enough to pass the e2e test. 
That's why I suggested to add a sleep before checking the instances, to ensure the behavior and avoid the risk of measuring the instance count during those seconds with 0 instances.
Have I explained myself now?

Reply:
yes, thanks for the explanation 😁